10611-finshes-part4

maimorag opened the pull request:
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/demisto/content?pullRequest=37970) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/demisto/content?pullRequest=37970) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/demisto/content?pullRequest=37970) it.</sub>

maimorag commented:
the validation fails since I deleted the image from the playbook folder (and moved it to the doc_files folder):
![image](https://github.com/user-attachments/assets/cda8d878-dee6-4a98-a7cf-1b8d32a18e18)

![image](https://github.com/user-attachments/assets/94c05d7f-9972-4971-9bb6-543e6d012392)
please force merge
